modules/vpc/vpc-flow-logs.tf

```diff
@@ -0,0 +1,9 @@
+# https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/flow_log
+
+resource "aws_flow_log" "default" {
+  count                = module.this.enabled && var.vpc_flow_logs_enabled ? 1 : 0
+  log_destination      = var.vpc_flow_logs_destination
```

Comment:
@aknysh @osterman I removed the usage of the `remote-state` module / file in this vpc component as it was only grabbing the `vpc-flow-logs-bucket` ARN from remote-state and it seemed to me to overly complicate things. I added the `vpc_flow_logs_destination` variable instead which means ya'll would need to add that as a var value in stacks. This is particularly simpler in the case where you do not want to enable VPC flow logs and / or create that flow logs bucket, which unfortunately is not supported because the `yaml-stack-config` module's `remote-state` module will err regardless of enabled or disabled. 

Anyway, let me know your thoughts on this one.